xml/cap_overview.xml

```diff
@@ -231,8 +232,10 @@
   </itemizedlist>
 
   <para>
-   &scf; packages upstream &cf; &bosh; releases to produce containers
-   and configurations which are deployed to &kube; clusters using &helm;.
+   &scf; uses &operator;, a &kube; Operator deployed via a &helm; chart, to
```

Comment:
`&scf; uses operator;,...` should `&scf;` be `&productname;` or `&cap;` here?

Reply:
My (probably wrong) understanding was that SCF will use KubeCF/quarks-operator and CAP packages SCF.